src/components/network-map-viewer/network/network-map.jsx

```diff
@@ -569,10 +686,30 @@ const NetworkMap = (props) => {
                 {showTooltip && renderTooltip()}
                 {/* visualizePitch true makes the compass reset the pitch when clicked in addition to visualizing it */}
                 <NavigationControl visualizePitch={true} />
+                <DrawControl
+                    position="bottom-left"
+                    displayControlsDefault={false}
+                    controls={{
+                        polygon: true,
+                        trash: true,
+                    }}
+                    //
+                    // defaultMode="simple_select | draw_polygon | ...
+                    defaultMode="simple_select"
```

Comment:
this is not defined in the interface `DrawControlProps`.
You pass `geoData` and `mapEquipments` and others ? into the `MapboxDraw` constructor ????
Maybe we could define a better API. Something like :
```
<DrawControl
  geoData={props.geoData}
  mapEquipments={props.mapEquipments}
  mapboxDrawParameters= {{
    position="bottom-left"
    displayControlsDefault={false}
    controls={{
        polygon: true,
        trash: true,
    }}
    ...
    />
}}
```